app/Notifications/CheckinAccessoryNotification.php

```diff
@@ -35,9 +37,13 @@ public function __construct(Accessory $accessory, $checkedOutTo, User $checkedIn
      */
     public function via()
     {
-        \Log::debug('via called');
         $notifyBy = [];
 
+        if (Setting::getSettings()->webhook_selected == 'microsoft'){
+
+            $notifyBy[] = MicrosoftTeamsChannel::class;
+        }
+
         if (Setting::getSettings()->webhook_endpoint != '') {
             $notifyBy[] = 'slack';
```

Comment:
This just caught my eye. I think we should be using the same check as above here (checking `webhook_selected` instead of `webhook_endpoint`). Right now I think this will end up being true whenever `webhook_selected === 'microsoft'` so `$notifyBy[]` will contain `MicrosoftTeamsChannel::class` and `slack`.

(We'll want to do that for all of the notifications)

Reply:
good catch, I changed it. I'm going to simplify this method in the next iteration